airflow/models/dataset.py

```diff
@@ -224,7 +224,7 @@ class DagScheduleDatasetAliasReference(Base):
         ForeignKeyConstraint(
             columns=(dag_id,),
             refcolumns=["dag.dag_id"],
-            name="dsdar_dag_fkey",
+            name="dsdar_dag_id_fkey",
```

Comment:
hmm? If the new name of contraint in migration is `dsdar_dag_fkey`, why are we changing this?